extension/content/script.js

```diff
@@ -353,6 +353,9 @@ async function fetchCached(url, expiry = ONE_DAY) {
   if (cache && Date.now() - expiry < cache.time) {
     return cache.data;
   }
+  if (url.endsWith("?_cachebust=")) {
+    url += Math.floor(Math.random() * 100000);
+  }
```

Comment:
Being pretty pedantic, but I this seems more brittle than it needs to be, something like 

   url = url.replace(/%CACHEBUST%/, Math.floor(Math.random() * 100000)); 

Means a little less confusion when we need to add more params on there? which itself seems unlikely but still

Reply:
Good point, I've made the change.